client/scss/components/_footer_nav.scss

```diff
@@ -1,3 +1,5 @@
+//* @define footer-nav
```

Comment:
You tell the linter what the B of BEM is, and it makes sure that you play by the rules (set out in the regex) from then on. Should prevent anyone new to the project/a cowboy sticking any old class in.